src/ubuntu/manifest.json

```diff
@@ -245,6 +245,20 @@
             }
           ]
         },
+        {
+          "platforms": [
+            {
+              "architecture": "amd64",
+              "dockerfile": "src/ubuntu/24.04/helix/android/amd64",
+              "os": "linux",
+              "osVersion": "noble",
+              "tags": {
+                "ubuntu-24.04-helix-android": {},
```

Comment:
We only want architecture-specific tags. Other tags without architecture are still around for backwards compatibility.

```suggestion
```